pjmedia/src/pjmedia-codec/g722.c

```diff
@@ -174,8 +183,8 @@ PJ_DEF(pj_status_t) pjmedia_codec_g722_init( pjmedia_endpt *endpt )
     g722_codec_factory.endpt = endpt;
     g722_codec_factory.pcm_shift = PJMEDIA_G722_DEFAULT_PCM_SHIFT;
 
-    g722_codec_factory.pool = pjmedia_endpt_create_pool(endpt, "g722", 1000, 
-						        1000);
+    g722_codec_factory.pool = pjmedia_endpt_create_pool(endpt, "g722", G722_POOL_INIT_SIZE, 
+						        G722_POOL_INC_SIZE);
```

Comment:
Sorry, forgot to mention this earlier.
- Perhaps better to put this setting in `pjmedia/include/pjmedia-codec/config.h`? Named `PJMEDIA_CODEC_POOL_LEN/INC_G722`?
- Also for all changes in the library code, please use max column length of 80.

Thank you.

Reply:
Putting a general define like `PJMEDIA_CODEC_POOL_LEN` in `pjmedia/include/pjmedia-codec/config.h` would imply, that every codec uses it, no?
If I introduce this define, should I update every codec impl to make use of it?
If every codec uses it, what would a good default value be?

Reply:
> Putting a general define like `PJMEDIA_CODEC_POOL_LEN` in `pjmedia/include/pjmedia-codec/config.h` would imply, that every codec uses it, no?

Was thinking for G722 only, as shown in the patch. But, having a common setting for all codecs sounds like a good idea.

> If I introduce this define, should I update every codec impl to make use of it?

Yes, unfortunately.

> If every codec uses it, what would a good default value be?

Assuming each codec instance will need to allocate about 100-500bytes (mostly for PJMEDIA codec structures, as some codec backends allocate their states by theirselves), IMO default value in the range of 1-5kB should be fine, and 2kB is perhaps reasonable as max calls in PJSUA by default is 4 (so 4*500B = 2kB).

Feel free if you have any opinions or suggestions.

Reply:
Moved and renamed the defines to pjmedia-codec's config.h now;
Having a define for every codec implementation allows for more fine tuning, if that should become necessary.

Reply:
Did you get a chance to look at the new changes?
I'm not sure whether I need to explicitly request a review or you get notified automatically.

Reply:
Yes, we get notified automatically. Sorry for the delay, must be in the middle of something when the notification came, but then forgot to come back to check it.

Things look okay to me, need to wait for another reviewer (from PJSIP team) to be able to merge.